sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -171,21 +171,55 @@ private async Task ProcessUntilCanceled(CancellationToken cancellationToken)
 }
 ```
 
+### Using an Active Directory principal with the Event Processor client
+
+The [Azure Identity library](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity/README.md) provides Azure Active Directory authentication support which can be used for the Azure client libraries, including Event Hubs and Azure Storage.
+
+To make use of an Active Directory principal, one of the available identity tokens from the `Azure.Identity` library is also provided when creating the Event Processor client.  In addition, the fully qualified Event Hubs namespace and the name of desired Event Hub are supplied in lieu of the Event Hubs connection string.
```

Comment:
Do you want "the" here? the fully qualified Event Hubs namespace and the name of `the` desired Event Hub ...

Reply:
Gah.   I had to read that three times before I realized what I did.